src/v/redpanda/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ v_cc_library(
     v::migrations
   DEFINES
     RPK_BINARY_PATH="${RPK_BINARY_PATH}"
+    HOST_PATH="$ENV{PATH}"
```

Comment:
Notes from tech call. Possible alternatives:
- May be able to use the path that RP is installed to 
- Considering the RPK deps are common, they likely exist in known system-wide path such as `/usr/bin` or `/usr/local/bin` so let's consider hard-coding those paths in
- The existence of these deps is not our responsibility